dallinger/command_line.py

```diff
@@ -296,26 +296,11 @@ def debug(verbose):
 
     # Try opening the psiTurk shell.
     try:
-        p = pexpect.spawn("psiturk")
-        p.expect_exact("]$")
-        p.sendline("server on")
-        p.expect_exact("Experiment server launching...")
-
-        # Launche the experiment.
-        time.sleep(4)
-
-        host = config.get("Server Parameters", "host")
-        port = config.get("Server Parameters", "port")
-
-        subprocess.call(
-            'curl --data "" http://{}:{}/launch'.format(host, port),
-            shell=True)
-
-        log("Here's the psiTurk shell...")
+        p = pexpect.spawn("python", [os.path.realpath(os.path.join(__file__, '..', 'heroku', 'psiturkapp.py'))])
```

Comment:
Since we're removing the ability to interact with the server via a shell, I don't think there's a need to use pexpect or run the server as a separate process. Am I forgetting something?